README.md

```diff
@@ -127,3 +129,8 @@ installed to run this:
  3. `python upload_parrots.py --team YOUR_TEAM --username YOUR_EMAIL --password YOUR_PASS`
 
  __Notice__: The uploader won't re-write already existing parrots if they match by name.
+
+## Discord via Mr. Parrot
+
+  1. Visit [this invite link](https://discordapp.com/oauth2/authorize?client_id=394830082058747905&permissions=1074006016&scope=bot) to add the bot to your server
+  2. Enjoy!
```

Comment:
This is all you have to do, right?

Reply:
The correct phrase would be "Click this link to invite the bot to your server"

Reply:
Yeah, what @justalemon is suggesting would fit more.